Apps/Examples/Examples/Assets.xcassets/AccentColor.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Why is this getting added here?

Reply:
Gets rid of a build warning about `AccentColor`.